diff -urN klook-2.0/src/audio.cpp klook-2.0-patched/src/audio.cpp
--- klook-2.0/src/audio.cpp	2013-02-22 00:33:26.000000000 +1100
+++ klook-2.0-patched/src/audio.cpp	2013-07-07 17:47:20.034443026 +1100
@@ -35,6 +35,7 @@
     , m_isPreview(false)
     , m_isReady(false)
 {
+    QTextCodec::setCodecForCStrings(QTextCodec::codecForName("UTF-8"));
     m_mediaObject = Phonon::createPlayer(Phonon::MusicCategory);
     m_mediaObject->setTickInterval(1000);
 
