--- klook-0.1/CMakeLists.txt~	2012-04-20 13:06:40.000000000 +0200
+++ klook-0.1/CMakeLists.txt	2012-06-27 22:10:55.813852489 +0200
@@ -41,7 +41,8 @@ target_link_libraries( klook ${KDE4_KIO_
  							${KDE4_PLASMA_LIBS}
 							${KDE4_KFILE_LIBS}
 							${QT_LIBRARIES}
-							${QT_QDECLARATIVE_LIBRARIES} )
+							${QT_QDECLARATIVE_LIBRARIES}
+						       	X11 )
 
 
 install( TARGETS klook ${INSTALL_TARGETS_DEFAULT_ARGS} )
